roles/addons/files/es-rc.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: elasticsearch-logging-v1
+  namespace: kube-system
+  labels:
+    k8s-app: elasticsearch-logging
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  replicas: 2
+  selector:
+    k8s-app: elasticsearch-logging
+    version: v1
+  template:
+    metadata:
+      labels:
+        k8s-app: elasticsearch-logging
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - image: gcr.io/google_containers/elasticsearch:1.7
+        name: elasticsearch-logging         
+        resources:
+          limits:
+            cpu: 100m
+        ports:
+        - containerPort: 9200
+          name: db
+          protocol: TCP
+        - containerPort: 9300
+          name: transport
+          protocol: TCP
+        volumeMounts:
+        - name: es-persistent-storage
+          mountPath: /data
+      volumes:
+      - name: es-persistent-storage
+        emptyDir: {}
```

Comment:
Sure, this is temporary solution

17 авг. 2015 г., в 17:49, Kenny Jones <notifications@github.com> написал(а):

In roles/addons/files/es-rc.yaml:

> +        resources:
> +          limits:
> +            cpu: 100m
> +        ports:
> +        - containerPort: 9200
> +          name: db
> +          protocol: TCP
> +        - containerPort: 9300
> +          name: transport
> +          protocol: TCP
> +        volumeMounts:
> +        - name: es-persistent-storage
> +          mountPath: /data
> +      volumes:
> +      - name: es-persistent-storage
> +        emptyDir: {}
I'm assuming the use of emptyDir is temporary, until storage is allocated on the worker nodes, and then it will then use storage that will persist longer?

—
Reply to this email directly or view it on GitHub.